source/installation-guide/wazuh-agent/index.rst

```diff
@@ -90,12 +90,23 @@ To install a Wazuh agent, select your operating system and follow the instructio
         </a>
     </div>
     <div class="item-agent">
-        <a href="./wazuh_agent_package_hpux.html" class="d-flex align-items-center">
+        <a href="./wazuh-agent-package-hpux.html" class="d-flex align-items-center">
             <p>HP-UX</p>
 
 .. image:: ../../images/installation/hpux.png
       :align: center
 
+.. raw:: html
+
+        </a>
+    </div>
+    <div class="item-agent" id="solaris-logo">
+        <a href="./wazuh-agent-package-solaris.html" class="d-flex align-items-center">
+            <p>Solaris</p>
+
+.. image:: ../../images/installation/solaris.png
+      :align: center
+
```

Comment:
The order of appearance of the operating systems has been changed in the new guide. The merge has caused the Solaris logo to be duplicated. 
![image](https://user-images.githubusercontent.com/61882981/144406463-fff4d680-9ca4-465e-83ff-127cc8512b3b.png)